packages/turf-rhumb-bearing/index.js

```diff
@@ -0,0 +1,60 @@
+// https://en.wikipedia.org/wiki/Rhumb_line
+// http://www.movable-type.co.uk/scripts/latlong.html#rhumblines
+var getCoords = require('@turf/invariant').getCoords;
+var GeodesyLatLon = require('geodesy').LatLonSpherical;
+
+/**
+ * Takes two {@link Point|points} and finds the bearing angle between them along a Rhumb line
+ * i.e. the angle measured in degrees start the north line (0 degrees)
+ *
+ * @name rhumbBearing
+ * @param {Feature<Point>} start starting Point
+ * @param {Feature<Point>} end ending Point
```

Comment:
Might be worth adding `Geometry Object` & `Array<number>` as valid input (input type supported when using `getCoord()`).

```javascript
* @param {Geometry|Feature<Point>|Array<number>} start starting Point
```